common/JackMidiPort.cpp

```diff
@@ -100,7 +100,7 @@ static void MidiBufferMixdown(void* mixbuffer, void** src_buffers, int src_count
     }
     mix->Reset(nframes);
 
-    uint32_t mix_index[src_count];
```

Comment:
This is both a memory leak and probably non-rt behavior in a place where you shouldn't have non-rt behavior. However, looking at the code, it seems like mix_index isn't really used! Seems like you can just delete this line and the line below referencing it.

Reply:
Ouch, forget that. My browser didn't mark all hits when searching. It is used. Need a different solution. alloca perhaps, or maybe there is a compilation option for vc to fix it.